tests/rbd/test_rbd_clone_flatten_group_snapshot.py

```diff
@@ -0,0 +1,284 @@
+                ):
+                    log.info("Parent details are reflected correctly for the cloned image of the group snapshot.")
+                else:
+                    log.error("Parent details are not reflected correctly for the cloned image of the group snapshot")
+                    return 1
+            else:
+                log.error("Parent option is not reflected in the clone info output")
+                return 1
+
+            # Map the cloned images as a new block disk using rbd map
+            # Mount the directory onto that disk and create some files on that directory and write data to the files
+            io_config["config"]["image_spec"] = [f"{pool}/{clone}"]
+
+            krbd_io_handler(**io_config)
+
+            # Flatten the cloned image using rbd flatten command.
+            flatten_kw = {"pool": pool, "image": clone}
+            out, err = rbd.flatten(**flatten_kw)
+            if out or err and "100% complete" not in err:
+                log.error(f"Flatten clone failed for {clone_spec}")
+                return 1
+            else:
+                log.info("Flattening clone successful")
+
+            # Verify the cloned image’s parent information after flattening using rbd info
+            out = rbd.info(**info_kw)
+            out_json = json.loads(out[0])
+
+            if "parent" in out_json:
+                log.error("parent information still exist for flattened clone")
+                return 1
+            else:
+                log.info(
+                    "Parent details do not exist after flattening clone as expected"
+                )
+
+            # write data to the flattened cloned image
+            krbd_io_handler(**io_config)
+            rm_kw = {"image-spec": f"{pool}/{clone}"}
+
+            # Remove the flattened cloned image using rbd rm command
+            out, err = rbd.rm(**rm_kw)
+            if out or err and "100% complete" not in err:
+                log.error(f"Remove flattened clone failed with error {err}")
+                return 1
+            else:
+                log.info("Flattened clone removed successfully")
+
+    return 0
+
+
+def run(**kw):
+    """
+    This test verifies clonign and flattening a group snapshot
+    Args:
+        kw: test data
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+
+    """
+    try:
+        log.info("CEPH-83594299 -  Cloning and Flattening a Group Snapshot")
+
+        if kw.get("client_node"):
+            client = get_node_by_id(kw.get("ceph_cluster"), kw.get("client_node"))
+        else:
+            client = kw.get("ceph_cluster").get_nodes(role="client")[0]
+        rbd_obj = initial_rbd_config(**kw)
+        pool_types = rbd_obj.get("pool_types")
+        if rbd_obj:
+            log.info("Executing test on Replication and EC pool")
+            if clone_flatten_group_snapshot(rbd_obj, client, **kw):
+                return 1
+
```

Comment:
add `log.info` for the successful message.

Reply:
Added